pyflakes/checker.py

```diff
@@ -1331,6 +1364,13 @@ def IMPORTFROM(self, node):
                                               module, alias.name)
             self.addBinding(node, importation)
 
+            if isinstance(self.scope, FunctionScope):
```

Comment:
This code block is the same as the block [above](https://github.com/PyCQA/pyflakes/pull/324/files#diff-f0429ba5bb9de6e83abcf687aca4c012R1328). It should probably be factored out into a separate function.

Reply:
yep